packages/augur-sdk/src/state/db/RollbackTable.ts

```diff
@@ -55,7 +55,7 @@ export class RollbackTable extends AbstractTable {
             const maxBlock =_.get(_.maxBy(documents, 'blockNumber'), 'blockNumber');
             await this.prune(maxBlock);
         }
-        await super.bulkPutDocumentsInternal(documents);
+        await super.bulkPutDocuments(documents);
```

Comment:
This will put this back in the queue. Its already gone through the queue though so seems it should still be internal